src/handlers/notify.rs

```diff
@@ -92,11 +92,18 @@ pub async fn handler(
 
         let id = chrono::Utc::now().timestamp_millis().unsigned_abs();
 
+        let topic = sha256::digest(&*hex::decode(data.sym_key)?);
+
+        info!(
+            "message to {}, on topic: {} with message_id: {}",
+            data.id, &topic, id
+        );
+
```

Comment:
can you add the `account` as well so it's easer to look up? and `client_id`?

Reply:
Data.id is `account`. Added `client_id`